### ISSUE TITLE
Who could run the code successfully?

### ISSUE THREAD

**Issue by rginpan** [wz0919/ScaleVLN#13]
Note that --use-env is set by default in torchrun.
If your script expects `--local-rank` argument to be set, please
change it to read from `os.environ['LOCAL_RANK']` instead. See 
https://pytorch.org/docs/stable/distributed.html#launch-utility for 
further instructions

  main()
usage: train_r2r.py [-h] [--vlnbert {cmt}] [--model_config MODEL_CONFIG] [--checkpoint CHECKPOINT] [--output_dir OUTPUT_DIR] [--train_batch_size TRAIN_BATCH_SIZE] [--val_batch_size VAL_BATCH_SIZE]
                    [--gradient_accumulation_steps GRADIENT_ACCUMULATION_STEPS] [--learning_rate LEARNING_RATE] [--valid_steps VALID_STEPS] [--log_steps LOG_STEPS] [--num_train_steps NUM_TRAIN_STEPS]
                    [--optim {adam,adamax,adamw}] [--betas BETAS [BETAS ...]] [--dropout DROPOUT] [--weight_decay WEIGHT_DECAY] [--grad_norm GRAD_NORM] [--warmup_steps WARMUP_STEPS] [--seed SEED] [--fp16]
                    [--n_workers N_WORKERS] [--pin_mem] [--local_rank LOCAL_RANK] [--node_rank NODE_RANK] [--world_size WORLD_SIZE] --config CONFIG
train_r2r.py: error: unrecognized arguments: --local-rank=0
